security/pkg/monitoring/monitoring.go

```diff
@@ -0,0 +1,29 @@
+package monitoring
+
+import "istio.io/pkg/monitoring"
+
+var (
+	RequestType = monitoring.MustCreateLabel("request_type")
```

Comment:
Could you please add a short comment for RequestType?

Reply:
Sure, this is copied from the old secretcache package by the way, so its now a new metric. Just moved to a common location.

Reply:
Added comment, thanks